lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_mock.c

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
the mock itself is now counted for coverage if it is under this folder.

Reply:
Good catch @ulaskozat ! I will move it to `test/mock_tasks`

Reply:
@ulaskozat , please review latest commit https://github.com/magma/magma/pull/10702/commits/32e6039a290b9fff35e4a9dcb31308f13a83fd74 for suggested changes